### PR TITLE
Support Bind9.10

### LINKED ISSUE #7: bind 9.10 exports differently?
Hi, I'm running this with bind 9.10: 1:9.10.3.dfsg.P2-4 under ubuntu, and there is barfs with:

```
ERRO[0028] Failed to unmarshal XML response: expected element type <isc> but have <statistics>  source=bind_exporter.go:219
```

### PR DESCRIPTION
**Pull request by miekg**
This is feeble attempt at a PR and totally not read to be merged, but it does support the new xml bind9.10 emits. It's working, but they've changed quite a lot. For server now emits:
```
bind_incoming_queries_total{code="NS",type="qtype"} 472
bind_incoming_queries_total{code="NSEC",type="qtype"} 154
bind_incoming_queries_total{code="Others",type="qtype"} 10
bind_incoming_queries_total{code="QUERY",type="opcode"} 10669
bind_incoming_queries_total{code="RESERVED10",type="opcode"} 0
bind_incoming_queries_total{code="RESERVED11",type="opcode"} 0
bind_incoming_queries_total{code="RESERVED12",type="opcode"} 0
bind_incoming_queries_total{code="RESERVED13",type="opcode"} 0
bind_incoming_queries_total{code="RESERVED14",type="opcode"} 0
```
The same could be per zone, as those structures are the same. Haven't look at the rest.
Added lame ass test and bind910 example XML file.

In short: don't merge :) It usable for me as-is.

Sort of fixes #7

### CODE REVIEW

**Comment by grobie** [comment]
Labels should only be used if it makes sense to sum up all values. Queries and requests should stay two separate metrics.

This seems to completely remove <9.10 support. We should be able to support both. At least as long major distributions don't use 9.10 by default.

File: bind_exporter.go [hunk 1]
```diff
@@ -214,33 +209,25 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
...
-	root := Isc{}
-	if err := xml.Unmarshal([]byte(body), &root); err != nil {
+	stats, err := unmarshal(body)
+	if err != nil {
 		log.Error("Failed to unmarshal XML response: ", err)
 		return
 	}
 
-	stats := root.Bind.Statistics
-	for _, s := range stats.Server.QueriesIn.Rdtype {
```
Comment (mattkanwisher) [inline bind_exporter.go]: Yeah looks like your removing a bunch of stats. Would probably be safe to just keep both sets

**Comment by miekg** [comment]
Well I am summing, but only for equal types so that should indeed be separated.

Yes, as I said: don't merge :) It could detect the xml on the fly and then do the right thing (tm). I don't have the time to make this into a proper pull request; please use the bits you can use and discard the rest.

**Comment by grobie** [comment]
Definitely thanks for contributing. I put it on my "open-source todo" list (it's long).

**Comment by miekg** [comment]
ghe ghe :-) I totally understand

**Comment by grobie** [comment]
Can be closed in favor of #11.

**Pull request closed without merging**